### PR TITLE
Topic2

### PR DESCRIPTION
**Pull request by DiegoJStroppiana**
Here is Topic 2 exercises

### CODE REVIEW

File: bootCampTopic2/bootCampTopic2ex1/src/test/java/maven/exercise/bootCampTopic2ex1/Exercise1Test.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	}
+	
+
+	
+	@Test	
+	public void openFile() throws IOException {
+		 System.out.println("");
+		 System.out.println("Test 2: Add File To Recent File List");	 
+		 String fileOpened;		
+		 File objetofile = new File ("Contacts.txt");
```
Comment (gastonabastoG) [inline bootCampTopic2/bootCampTopic2ex1/src/test/java/maven/exercise/bootCampTopic2ex1/Exercise1Test.java:32]: What do you do with this variable?

File: bootCampTopic2/bootCampTopic2Ex2/src/test/java/maven/exercise/bootCampTopic2Ex2/BlogTest.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		
+	}
+
+
+	@Test	
+	public void addEntry() throws IOException {	
+		 System.out.println("");
+		 System.out.println("Test 1: Post new entry");
+		 Blog.getInstance().newEntry(new Entry ("New entry","This is a new entry"));
+		 Blog.getInstance().showRecentEntries();
```
Comment (gastonabastoG) [inline bootCampTopic2/bootCampTopic2Ex2/src/test/java/maven/exercise/bootCampTopic2Ex2/BlogTest.java:41]: Actually to be a good test you must use assert sentence (and also could be add verify)

File: bootCampTopic2/bootCampTopic2Ex2/src/test/java/maven/exercise/bootCampTopic2Ex2/BlogTest.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	}
+	
+	@Test	
+	public void deleteEntry() throws IOException {		
+		 System.out.println("");
+		 System.out.println("Test 2: Delete existing entry");
+		 Blog.getInstance().deleteEntry("Title 5");
+		 Blog.getInstance().showRecentEntries();
```
Comment (gastonabastoG) [inline bootCampTopic2/bootCampTopic2Ex2/src/test/java/maven/exercise/bootCampTopic2Ex2/BlogTest.java:51]: Actually to be a good test you must use assert sentence (and also could be add verify)

File: bootCampTopic2/bootCampTopic2Ex2/src/test/java/maven/exercise/bootCampTopic2Ex2/BlogTest.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+	}
+	
+	@Test
+	public void showRecentEntries() throws IOException{
+		 System.out.println("");
+		 System.out.println("Test 3: Show recent entries");
+		 Blog.getInstance().showRecentEntries();
```
Comment (gastonabastoG) [inline bootCampTopic2/bootCampTopic2Ex2/src/test/java/maven/exercise/bootCampTopic2Ex2/BlogTest.java:59]: Actually to be a good test you must use assert sentence (and also could be add verify)

File: bootCampTopic2/bootCampTopic2ex1/src/test/java/maven/exercise/bootCampTopic2ex1/Exercise1Test.java (hunk 1 above)
Comment (DiegoJStroppiana) [reply bootCampTopic2/bootCampTopic2ex1/src/test/java/maven/exercise/bootCampTopic2ex1/Exercise1Test.java:32]: I created this variable to try to create a file and get its name to add to the list

**Pull request merged**